analyzers/src/SonarAnalyzer.CSharp/Rules/ClassShouldNotBeAbstract.cs

```diff
@@ -47,13 +47,13 @@ public sealed class ClassShouldNotBeAbstract : SonarDiagnosticAnalyzer
 
                     if (!IsRecordWithParameters(symbol) && AbstractTypeShouldBeInterface(symbol))
                     {
-                        Report(symbol, MessageToInterface, c);
+                        Report(c, symbol, MessageToInterface);
```

Comment:
would be great to rename `c` to `symbolContext` or similar.

Reply:
Not, here, as it's the usual lambda parameter